language/automl/README.md

```diff
@@ -0,0 +1,88 @@
+# AutoML Sample
+
+<a href="https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/java-docs-samples&page=editor&open_in_editor=vision/beta/cloud-client/README.md">
+<img alt="Open in Cloud Shell" src ="http://gstatic.com/cloudssh/images/open-btn.png"></a>
+
+[Google Cloud Natural Language API][language] provides feature detection for images.
```

Comment:
This API is for AutoML Natural Language, not for the Natural Language API in general.

Also, it's not for detecting images